web/static/js/Install.jsx

```diff
@@ -67,9 +67,14 @@ var VimPlugInstructions = React.createClass({
 // Instructions for installing a plugin with Pathogen.
 var PathogenInstructions = React.createClass({
   render: function() {
+    //if the url has a trailing slash remove it
+    var url = (this.props.github_url || "").replace(/\/$/, "");
+    // now get the words after last / in url (the title)
+    var pos = url.lastIndexOf("/") + 1;
+    var bundleName = url.substr(pos);
```

Comment:
Rather than doing this string parsing here, I think it would be preferable to pass the `github_repo_name` prop from `Plugin` through to this component.